contracts/pool/src/contract.rs

```diff
@@ -535,17 +537,17 @@ impl LiquidityPoolTrait for LiquidityPool {
         }
     }
 
-    fn simulate_swap(env: Env, sell_a: bool, offer_amount: i128) -> SimulateSwapResponse {
+    fn simulate_swap(env: Env, offer_asset: Address, offer_amount: i128) -> SimulateSwapResponse {
+        let config = get_config(&env);
+
         let pool_balance_a = utils::get_pool_balance_a(&env);
         let pool_balance_b = utils::get_pool_balance_b(&env);
-        let (pool_balance_offer, pool_balance_ask) = if sell_a {
+        let (pool_balance_offer, pool_balance_ask) = if offer_asset == config.token_a {
```

Comment:
wasn't this change and similar already introduced via [141](https://github.com/Phoenix-Protocol-Group/phoenix-contracts/issues/141)

Reply:
that was done in Pair, nvm